tasks/main.yml

```diff
@@ -5,23 +5,148 @@
 - name: Set platform/version specific variables
   include_tasks: tasks/set_vars.yml
 
-# Examples of some tasks:
-- name: Ensure required packages are installed
+- name: Check if requested version is supported in the system (RHEL8)
+  fail:
+    msg: "RHEL 8 supports only Postgresql 10, 12 and 13"
+  when:
+    - ansible_facts['os_family'] == "RedHat"
```

Comment:
prefer to use double quotes consistently - unless in a jinja construct where you have to use single quotes e.g.
`somevar: "{{ prefix ~ '/' ~ suffix }}"`